plugins/CoreHome/angularjs/reporting-page/reportingpage.directive.html

```diff
@@ -4,6 +4,8 @@
 
     <div ng-show="hasNoPage">{{ 'CoreHome_NoSuchPage'|translate }}</div>
 
+    <div class="alert alert-info" ng-show="showOnlyRawDataAlert" ng-bind-html="rawDataMessage"></div>
```

Comment:
Is there any reason for directly having the alert in the code and not using the notification api? See
https://github.com/matomo-org/matomo/blob/115527353a9e75e01aa4d263408956ae45403bea/plugins/GeoIp2/angularjs/geoip2-updater/geoip2-updater.controller.js#L93-L100

Reply:
I know it was suggested that way in the original issue, but imho the grey on grey doesn't look good:

![image](https://user-images.githubusercontent.com/1579355/112472081-fddc7500-8d6c-11eb-9aff-68f90d087161.png)

Reply:
The notification is different than the alert, other css, divs, etc. @tsteur mentioned this alert, because this one has this gray version. 
And I know we have actually an alert angularjs component as well, but I had issues with the link inside the notification (it is sanitized), so I just used the native div+css.

Reply:
I just talked to @tsteur, will use the notification.